messenger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .settings import *
```

Comment:
The purpose of turning messenger info a module was whaf I felt a natural evolution of the code base. I wanted go separate files to help manage dependencies, keep code DRY and hook into the setuptools tool chain to manage complication

Currently the entire module can be run as a script. Whereas in the past you would run

    cd messenger
    python make.py matlab
    cd ..

You would now just run

python -m messenger matlab